App.js

```diff
@@ -8,12 +8,19 @@
 
 import React, { Component } from 'react';
 import Navigator from './Navigation/Navigation';
+import { fillBasketDB } from './Database/prefillDB';
 
 
 export default class App extends Component {
-  render() {
-    return (
-      <Navigator />
-    );
-  }
+
+    componentDidMount() {
+        // uncomment the following line to prefill basket and product database
+        // fillBasketDB();
```

Comment:
Uncomment to prefill databases

Reply:
C'est partique ce script de remplissage !